spring-data-rest-hal-browser/src/main/resources/META-INF/spring-data-rest/hal-browser/js/CustomPostForm.js

```diff
@@ -44,6 +44,11 @@ var CustomPostForm = Backbone.View.extend({
 			data: this.getNewResourceData()
 		};
 
+		var cookie = document.cookie.match('(^|;)\\s*' + 'XSRF-TOKEN' + '\\s*=\\s*([^;]+)');
+		if(cookie){
```

Comment:
I'd put the space between `if` and `(cookie)` to be consistent with the rest of the code.